LandSandBoat/server
# [sql] Add Bracers de-synthesis recipe

**_I affirm:_**

## What does this pull request do?

Adds desynth. recipe for Bracers

Source: https://ffxiclopedia.fandom.com/wiki/Bracers#Desynthesis_Recipes

Clothcraft (25/36)
Yield: Saruta Cotton (**834**) x 2
HQ 1: Linen Thread (**819**) x 5
HQ 2: Linen Thread (**819**) x 6
HQ 3: Linen Thread (**819**) x 7
Lightning Crystal (**4100**)
1 x Bracers (**12722**)

Item IDs (aside for the craft skill level range) are in the parentheses.  Recipe was added to FFXIclopedia back in 2014 by author of other desynth. recipes and also a desynth. guide.

## Steps to test these changes

- !additem 12722
- !additem 4100
- !setcraftrank 52 9
- !setskill 52 100
- Attempt to desynth. Bracers before this PR, see your character say that it can't do that.
- Attempt to desynth. Bracers after this PR, see your character **shocked** by the desynth. that occurs (**shocked** because lightning crystal used for the desynth......get it?.....get it?.....I'll show myself back to my corner now).
